### ISSUE TITLE
'/var/host/shill/resolv.conf': No such file or directory

### ISSUE THREAD

**Issue by rackstraw** [dnschneid/crouton#3939]
```
$ sudo edit-chroot -all

name: xenial
encrypted: no
crouton: version 1-20171122220410~master:99877eaf
release: xenial
architecture: amd64
xmethod: xorg
targets: gnome,xorg,cli-extra
host: version 9901.77.0 (Official Build) stable-channel celes 
kernel: Linux localhost 3.18.0-16037-gf59ef0b48a68 #1 SMP PREEMPT Mon Nov 13 16:34:35 PST 2017 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
```

#### Please describe your issue:
All attempts to update chroot failed.  None of the fixes given in issue #3555  worked so I aborted the update, and attempted to enter the original chroot.

#### If known, describe the steps to reproduce the issue:
$ sudo startcli
Entering /mnt/stateful_partition/crouton/chroots/xenial...
A chroot setup script still exists inside the chroot.
The chroot may not be fully set up.
Would you like to finish the setup? [Y/n/d] d
Removing the chroot setup script. You may want to update your chroot again.
ln: failed to access '/var/run/crouton/mnt/stateful_partition/crouton/chroots/xenial/etc/resolv.conf': Permission denied
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...

$ sudo startcli
Entering /mnt/stateful_partition/crouton/chroots/xenial...
error: failed to access '/var/run/crouton/mnt/stateful_partition/crouton/chroots/xenial/etc/resolv.conf': Permission denied
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...

$ sudo ls -l /var/run/crouton/mnt/stateful_partition/crouton/chroots/xenial/etc/resolv.conf
lrwxrwxrwx.  1 root root     27 Apr 15  2018 resolv.conf -> /var/host/shill/resolv.conf

$ sudo ls -l /var/host/shill/resolv.conf
ls: cannot access '/var/host/shill/resolv.conf': No such file or directory

Bottom line, I need to replace a missing file in /var/host/shill in order to boot. How do I do it?

**Comment by DennisLfromGA** [comment]
@rackstraw,

Hmm, not sure why it's failing.
I did the following from inside the chroot, it looks pretty similar to yours except of course for the file `/var/host/shill/resolv.conf` -

```
(trusty)denny@localhost:~$ ll -d /etc/resolv.conf 
lrwxrwxrwx. 1 root root 27 Nov 13 13:18 /etc/resolv.conf -> /var/host/shill/resolv.conf
(trusty)denny@localhost:~$ 
(trusty)denny@localhost:~$ ll /etc/resolv.conf 
lrwxrwxrwx. 1 root root 27 Nov 13 13:18 /etc/resolv.conf -> /var/host/shill/resolv.conf
(trusty)denny@localhost:~$ 
(trusty)denny@localhost:~$ ll /var/host/shill/resolv.conf 
-rw-r--r--. 1 root root 82 Nov 16 10:39 /var/host/shill/resolv.conf
(trusty)denny@localhost:~$ 
(trusty)denny@localhost:~$ cat /var/host/shill/resolv.conf 
nameserver 8.8.4.4
nameserver 8.8.8.8
options single-request timeout:1 attempts:5
(trusty)denny@localhost:~$ 
```

I listed the contents of `/var/host/shill/resolv.conf` if you want to create it and see if that works for you.

*Hope this helps,*
-**DennisLfromGA**

**Comment by DennisLfromGA** [comment]
Closing. No response or follow-up (needed) at this time.

-**DennisLfromGA**
